app/assets/stylesheets/hackweek.css.scss

```diff
@@ -45,15 +61,131 @@ footer {
 }
 
 .project-list-item:before {
-    display:block;
-    position: relative;
-    content:"";
-    height:70px;
-    margin:-70px 0 0;
-    z-index: -1;
+  display:block;
+  position: relative;
+  content:"";
+  /* height:70px;
+   margin:-70px 0 0;*/
+  z-index: -1;
 }
 
 #loading-spinner {
   text-align: center;
   vertical-align: bottom;
 }
+
+.well {
```

Comment:
Please do not change the layout of bootstrap components. I guess you want to target a specific well, add another class please.